### PR TITLE
Added Laravel 5.4 compatibility

### LINKED ISSUE #307: Compatibility Laravel 5.4
version tested 1.10.0

On composer update, during optimize I get: 

Call to undefined method Illuminate\Session\Store::getToken()

### PR DESCRIPTION
**Pull request by mauricius**
Fixes #307

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     },
     "require-dev": {
         "phpunit/phpunit": "~4.0",
-        "orchestra/testbench": "~3.0"
+        "orchestra/testbench-browser-kit": "~3.4"
```
Comment (kristijanhusak) [inline composer.json]: Why this needs to be changed? Looks like testbench repo has support for 5.4.
https://github.com/orchestral/testbench

Comment (mauricius) [inline composer.json]: You are right! Some tests failed using version 3.4 of testbench, but they passed using browser-kit. Probably I just needed to update phpunit to the latest version.

Comment (mauricius) [inline composer.json]: I have restored the original testbench package. Turned out that the interaction with session functionality has been moved to testbench-browser-kit, so I decided to use the `InteractsWithSession` trait directly where required.

**Comment by kristijanhusak** [comment]
Thanks @mauricius , it looks great.

File: src/Kris/LaravelFormBuilder/Form.php [hunk 2]
```diff
@@ -3,7 +3,7 @@
 use Illuminate\Contracts\Events\Dispatcher as EventDispatcher;
 use Illuminate\Contracts\Validation\Factory as ValidatorFactory;
 use Illuminate\Contracts\Validation\Validator;
-use Illuminate\Http\Exception\HttpResponseException;
+use Illuminate\Http\Exceptions\HttpResponseException;
 use Illuminate\Http\Request;
```
Comment (barryvdh) [inline src/Kris/LaravelFormBuilder/Form.php]: Breaks pre-5.4

**Pull request merged**